src/newgrf.cpp

```diff
@@ -2144,6 +2171,13 @@ static ChangeInfoResult StationChangeInfo(uint stid, int numinfo, int prop, Byte
 				AddStringForMapping(buf->ReadWord(), [statspec](StringID str) { StationClass::Get(statspec->cls_id)->name = str; });
 				break;
 
+			case 0x1E: { // Extended tile flags (replaces prop 11, 14 and 15)
+				uint16_t tiles = buf->ReadExtendedByte();
```

Comment:
Should this limit the amount of elements to just 256? Or at least throw a warning of some sort?
After all, how much sense does it make to have thousands of tile flags when only the first 256 can be used?